### PR TITLE
More item calls have been replaced with the new functions

### PR DESCRIPTION
**Pull request by annando**
Attention: PR https://github.com/friendica/friendica-addons/pull/621 has to be merged first.

This PR removes legacy item functions and it removes a bug where we have to use the user contact's avatar with feeds.

### CODE REVIEW

File: mod/ping.php [hunk 1]
```diff
@@ -127,20 +128,14 @@ function ping_init(App $a)
...
-				 `item`.`deleted` = 0 AND `item`.`uid` = %d AND `pitem`.`parent` != 0
-				AND `item`.`contact-id` != %d
-				ORDER BY `item`.`created` DESC",
-			intval(local_user()),
-			intval(local_user())
-		);
+		$condition = ["`unseen` AND `uid` = ? AND `contact-id` != ?", local_user(), local_user()];
+		$fields = ['id', 'parent', 'verb', 'author-name', 'unseen', 'author-link', 'author-avatar', 'contact-avatar',
+			'network', 'created', 'object', 'parent-author-name', 'parent-author-link', 'parent-guid'];
+                $params = ['order' => ['created' => true]];
```
Comment (MrPetovan) [inline mod/ping.php]: Coding standards: Wrong indentation style.

Comment (annando) [reply mod/ping.php]: Done.

File: mod/ping.php [hunk 2]
```diff
@@ -127,20 +128,14 @@ function ping_init(App $a)
...
-				AND `item`.`contact-id` != %d
-				ORDER BY `item`.`created` DESC",
-			intval(local_user()),
-			intval(local_user())
-		);
+		$condition = ["`unseen` AND `uid` = ? AND `contact-id` != ?", local_user(), local_user()];
+		$fields = ['id', 'parent', 'verb', 'author-name', 'unseen', 'author-link', 'author-avatar', 'contact-avatar',
+			'network', 'created', 'object', 'parent-author-name', 'parent-author-link', 'parent-guid'];
+		$params = ['order' => ['created' => true]];
+                $items = Item::select(local_user(), $fields, $condition, $params);
```
Comment (MrPetovan) [inline mod/ping.php]: Coding standards: Wrong indentation style. I thought it would be fixed as well with the previous line! 😅

Comment (annando) [reply mod/ping.php]: Done.

**Comment by MrPetovan** [comment]
Follow-up to #5184

**Pull request merged**